hooks/playbooks/service-project-more-quota.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Increase quotas for the service project
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Set the volumes to 50
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.shell: |
+        oc project {{ namespace }}
+        oc rsh openstackclient \
+           openstack quota set --volumes 50 service
```

Comment:
Thank you, this is something I wanted to implement. I didn't move forward because I wasn't sure whether I can assume the name 'service' is kind of hardcoded in operator-based OSP deployments, or it should be a parameter, or it should be gathered from some source dynamically.

Reply:
In today's code this is [hardcoded in the cinder-operator](https://github.com/openstack-k8s-operators/cinder-operator/blob/8f627d558478e8e5b92177bf18db5eb2f32cbfba/templates/cinder/config/00-global-defaults.conf#L94):

```ini
[service_user]
send_service_user_token = True
auth_url = {{ .KeystoneInternalURL }}
auth_type = password
project_domain_name = Default
user_domain_name = Default
project_name = service
```